### ISSUE TITLE
Move to JavaFX

### ISSUE THREAD

**Issue by Arc676** [Arc676/Toohak#7]
Switching to JavaFX instead of Swing might solve the Windows font size issue, where all text in labels and buttons are rendered smaller on Windows due to different font size calculations.

**Comment by MarekKristof** [comment]
Hello, can i take it ?

**Comment by Arc676** [comment]
Of course! Thank you for your interest in the project. You're more than welcome to fork and make a PR; I'd appreciate it!

**Comment by MarekKristof** [comment]
Hello, i would like to ask if you can add someone like second contributor,
because i would like do your project like school project, but condidion is,
that project need to have min 2 contributors.

ne 14. 10. 2018 o 10:16 Alessandro Vinciguerra <notifications@github.com>
napísal(a):
